test/e2e/specs/plans/plans__signup-business.ts

```diff
@@ -29,7 +29,7 @@ describe(
 		let testAccount: TestAccount;
 		let page: Page;
 		let siteCreatedFlag = false;
-		let siteDetails: SiteDetails;
+		let newSiteDetails: NewSiteResponse;
```

Comment:
Changes in this file are precipitated by changes to the return type of `SignupPickPlanPage.selectPlan`.